client/src/components/board-details/BoardPermissions/BoardPermissions.tsx

```diff
@@ -86,9 +87,12 @@ export const BoardPermissions = () => {
               <TableCell className="text-foreground">{user.addedAt}</TableCell>
               <TableCell className="text-foreground">{user.invitedBy}</TableCell>
               <TableCell className="flex items-center justify-center">
-                <Button variant="outline" className="hover:text-muted-foreground">
-                  <TrashIcon className="h-4 w-4" />
-                </Button>
+                <DeleteCollaboratorButton
+                  username={user.name}
+                  deleteUser={() => {
+                    /*todo: implement collaborator deletion*/
```

Comment:
Can you add a task on Jira for this? It sound like a nice stand-alone task candidate.

Reply:
done